pandas/tests/indexes/interval/test_interval_range.py

```diff
@@ -84,7 +84,7 @@ def test_constructor_timestamp(self, closed, name, freq, periods, tz):
         tm.assert_index_equal(result, expected)
 
         # GH 20976: linspace behavior defined from start/end/periods
-        if not breaks.freq.is_anchored() and tz is None:
+        if not breaks.freq.n == 1 and tz is None:
             # matches expected only for non-anchored offsets and tz naive
```

Comment:
comment here refers to anchoring.  i suspect the comment (which i suspect i wrote) misunderstood what "is_anchored" really meant

Reply:
thanks, I removed the comment